docs/release-notes/Percona-Everest-1.3.0-(2024-10-28).md

```diff
@@ -0,0 +1,228 @@
+
+- [EVEREST-1211](https://perconadev.atlassian.net/browse/EVEREST-1211) - You can now easily edit your resources directly from the **Overview** page. There’s no longer a need to navigate the entire database wizard, saving you time and simplifying the process.
+
+- [EVEREST-1459](https://perconadev.atlassian.net/browse/EVEREST-1459) - We have added a link to **Percona Support** on the Percona Everest home page, making it easier for you to contact support if needed.
+
+
+- [EVEREST-1460](https://perconadev.atlassian.net/browse/EVEREST-1460) - To make your experience with Percona Everest even smoother, we've added convenient links right on the login page. Discover everything from **Support** and a **Quickstart guide** to our **Forum**, the **K8s Squad program**, and our **GitHub** repository.
+
+
+- [EVEREST-1470](https://perconadev.atlassian.net/browse/EVEREST-1470) - The `rbac validate` command has been enhanced to accept the `ConfigMap` YAML file. This enables you to validate role-based access control (RBAC) configurations by leveraging the structured data provided in a `ConfigMap` format.
+
+- [EVEREST-1533](https://perconadev.atlassian.net/browse/EVEREST-1533) - Users with read-only permissions for a namespace, including all database engines and database clusters within that namespace, currently cannot access the **Upgrade** option in the user interface. This restriction prevents them from viewing upgrade prerequisites, such as the versions of database clusters that may need to be upgraded.
+
+    However, starting with Percona Everest 1.3.0, the **Upgrade** button is clickable for these users. This enables them to view details about the upgrade plan, including any necessary changes for the database clusters, which can help inform administrators about required preparations. However, the option to upgrade the operator remains unclickable for users without the upgrade permissions.
+
+## Deprecated API endpoints
+
+This is the list of the API endpoints deprecated in Percona Everest v1.2.0 and removed from v1.3.0:
+
+
+|**No**|**API endpoints**|**Method**|
+|---------|--------------|------------|
+|**a.**|`/monitoring-instances`|1.`GET`</br> 2.`POST`|
+|**b.**|`/monitoring-instances/{name}`|1.`GET`</br>2. `PATCH`</br>3.`DELETE`|
+|**c.**|`/backup-storages`|1.`GET`</br> 2.`POST`|
+|**d.**|`/backup-storages/{name}`|1.`GET`</br>2. `PATCH`</br>3.`DELETE`|
+
+
+## Bugs
+
+- [EVEREST-886](https://perconadev.atlassian.net/browse/EVEREST-886) - The PostgreSQL database can now be restored using PITR. Previously, the restoration using the latest PITR did not complete, and the database remained in the **Restoring** status.
```

Comment:
We need to delete the mention of this ticket since as it turned out it wasn't fixed and we returned to the old implementation with heuristics

Reply:
Fixed